devtools/create_achievements/create_achievements.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+
+#AGS games:
+python steam_achievements.py -v --steamid 80310
```

Comment:
This should get added to a function. It reduces duplication but more importantly it adds the ability to perform error checking and error reporting in a single location. The exit codes of `steam_achievements.py` should get checked here - see my comment below

Reply:
Introduced functions at 7b46c8b1b5b8780d94164d2c08810e932cf5b845
No need to check $? in them since script works at "set -e" mode now.